lib/secret/secret.go

```diff
@@ -108,6 +108,16 @@ func (k Key) Open(ciphertext []byte) ([]byte, error) {
 		return nil, trace.Wrap(err)
 	}
 
+	// DELETE IN: 4.1.0
```

Comment:
Should this check actually be deleted in `4.1.0`?  Since JSON decoding occurs in the body of the function, it seems reasonable to assume that the caller may not have verified nonce size before calling `Open`.
If this is the desired behavior, maybe the `secret` package should document that it is unsuitable for use against untrusted input?

Reply:
Fair point, removed the comment.